ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsListPage.tsx

```diff
@@ -70,7 +69,8 @@ function IntegrationsListPage({
     const isScannerV4 = getIsScannerV4(source, type);
     const isCloudSource = getIsCloudSource(source);
 
-    const isTechPreview = isFeatureFlagEnabled('ROX_SCANNER_V4') && type === 'scannerv4';
+    // There is currently nothing relevant in Tech Preview.
+    const isTechPreview = false;
```

Comment:
Same here. You can remove the `isTechPreview` variable and the sections using it